### ISSUE TITLE
[a11y] Template should allow configurable alt text for cover images

### ISSUE THREAD

**Issue by shinytoyrobots** [shinytoyrobots/shinytoyrobots-hugo#4]
Currently the hello-friend theme uses the post title for the cover image alt text. That isn't really accessible - it should be descriptive of the image itself.

Customize theme and/or submit change back to [theme repo](https://github.com/panr/hugo-theme-hello-friend/).

**Comment by shinytoyrobots** [comment]
Closed by https://github.com/shinytoyrobots/hugo-theme-hello-friend/pull/1
